playbooks/openstack/openshift-cluster/launch.yml

```diff
@@ -55,7 +55,7 @@
     register: stack_show_status_result
```

Comment:
I 100% agree with your point @detiber !

Unfortunately, it was not possible to ask `heat stack-create` to be synchronous in version 0.4.0. The latest version I just tried (0.8.0) learnt the `--poll` switch which aims at making `heat stack-create` synchronous.
I tried it and with this option, `heat stack-create --poll` never returns. It seems there is an issue in the exit condition. So, it is not usable.

Moreover, in heat 0.8.0, the `--poll` switch exists only on `stack-create` and not on `stack-delete` so that there are still no way to make the stack deletion synchronous.